reference/shaders-hlsl/frag/basic.frag

```diff
@@ -0,0 +1,31 @@
+uniform sampler2D uTex;
```

Comment:
There's different ways to handle textures - no expert on that, sampler2D decls work good enough for me. And yes, no layouts currently, apllications have to set up input element descriptions accordingly.